crates/bevy_render/src/renderer/render_device.rs

```diff
@@ -129,6 +132,19 @@ impl RenderDevice {
         Texture::from(wgpu_texture)
     }
 
+    /// Creates a new [`Texture`] and initializes it with the specified data.
+    ///
+    /// `desc` specifies the general format of the texture.
+    pub fn create_texture_with_data(
```

Comment:
I added this over here:
* https://github.com/bevyengine/bevy/pull/3884
* https://github.com/bevyengine/bevy/pull/3884/files#diff-a1595e1935032ef7580c1c4530e830f43efb3ba0e6c2de3aa2b3ffe788729091R130

Reply:
That got merged so please update your PR on main.

Reply:
Done. :smile: